db/seeds.rb

```diff
@@ -61,3 +61,15 @@
   communal_area: communal_area,
   priority: [priority1, priority2, priority3, priority4].sample
 )
+
+user = FactoryBot.create(:user)
+
+Defect.all.each do |defect|
```

Comment:
Could we maybe document how this differs from the normal behaviour in the commit message and as a comment?